strobogrammatic-and-palindrome.html

```diff
@@ -1,74 +1,87 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="dark">
```

Comment:
Instead of one theme, let's have a toggle switch so users can toggle between their preferred themes. Read about dark mode support in TailwindCSS. Ref: https://tailwindcss.com/docs/dark-mode

Reply:
It won't be possible as we'd need to configure an NPM project and have a `tailwind.config.js` file for configuration. My bad!